webapp/src/components/tutorialValidators.tsx

```diff
@@ -1,19 +1,17 @@
 import { MarkedContent } from "../marked";
 import { getBlocksEditor } from "../app";
-import TutorialOptions = pxt.tutorial.TutorialOptions;
-import TutorialStepInfo = pxt.tutorial.TutorialStepInfo;
 import CodeValidator = pxt.tutorial.CodeValidator;
 import CodeValidatorMetadata = pxt.tutorial.CodeValidatorMetadata;
 import CodeValidationResult = pxt.tutorial.CodeValidationResult;
 import CodeValidationExecuteOptions = pxt.tutorial.CodeValidationExecuteOptions;
 
-const defaultResult: CodeValidationResult = {
+const defaultResult: () => CodeValidationResult = () => ({
     isValid: true,
     hint: null,
-};
+});
 
 export function GetValidator(metadata: CodeValidatorMetadata): CodeValidator {
```

Comment:
Might still move these over and split up later on, but as mentioned in description putting that off till we have a few more use cases to see how much we need to reuse it.